src/e2e/specs/dashboard.spec.ts

```diff
@@ -302,6 +302,58 @@ test.describe(`${process.env.E2E_TEST_ENV} - Breaches Dashboard  - Payment`, ()
       `${process.env.E2E_TEST_ENV === "prod" ? "yearly" : "every ⁨2⁩ months"}`,
     );
   });
+
+  test("Verify that the Premium upsell screen is displayed correctly - overview card", async ({
+    dashboardPage,
+    // purchasePage,
+    page,
+  }) => {
+    test.info().annotations.push({
+      type: "testrail",
+      description:
+        "https://testrail.stage.mozaws.net/index.php?/cases/view/2463625",
+    });
+
+    //checking that the user can reach upsell page
+    await dashboardPage.goToDashboard();
+    const upsellButton = page.locator("section > div a");
+    await expect(upsellButton).toBeVisible();
+    await upsellButton.click();
+    await page.waitForURL(/.*\/fix\/.*\/view-data-brokers\/?/);
+    const removeThemButton = page.getByText("Remove them for me");
+    await removeThemButton.click();
+
+    //checking the bullet points
+    await page.waitForURL(/.*\/fix\/.*\/automatic-remove\/?/);
+    const ulElement = page.locator("div > strong + ul");
+    expect(await ulElement.isVisible()).toBeTruthy();
+
+    const expectedItems = [
+      "Monthly scan of ⁨190⁩ data broker sites that may be selling your personal info",
+      "Automatic data removal from sites that are selling your personal info",
+      "Guided experience through high risk data breaches that require manual steps",
+      "Continuous monitoring for new exposures",
+      "Alerts when your data has been breached",
+    ];
+
+    for (const itemText of expectedItems) {
+      const liElement = page.locator(`ul li:has-text("${itemText}")`).first();
+      await expect(liElement).toBeVisible();
+    }
+
+    //testing that toggles work
+    const label0 = page.locator("div > label").nth(0);
+    const label1 = page.locator("div > label").nth(1);
+
+    await label0.click();
+    const price0 = page.locator("div > strong + span").textContent();
```

Comment:
We should put these locators in one of the `page` files, probably dashboardPage.ts in this case.

I believe we can re-use one of the existing properties from the class. 

If not, instead of using an xpath, using something like `page.getByText('Yearly', { exact: true })` should work for the yearly or monthly toggle buttons.

Reply:
@rafeerahman Do you think it should be in dashboardPage.ts if they're multiple clicks away from being visible?

Reply:
I think creating a new page file would be the best option. Something like automaticRemovalPage.ts? You can follow the outline of the dashboardPage.ts class.

When you create a new page file, you also need to add it to /fixtures/basePages.ts so you can use it within the spec files.